common/adminBlock/EntryAddReplaceMatryoshkaHash.go

```diff
@@ -10,8 +10,11 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
+// AddReplaceMatryoshkaHash is a DEPRECATED admin block entry that adds a Matryoshka Hash to the specified server.
+// According to 'Who', the Matryoshka hash is not used any longer, although its unclear if it may make an appearance
+// in older sections of the block chain.
```

Comment:
added, the tricky piece with these were the lack of comments and overall lack of documentation on them, I couldn't add too many details, because I didn't fully understand them